blueprints/app/files/.prettierrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   plugins: ['prettier-plugin-ember-template-tag'],
   overrides: [
     {
-      files: '*.{js,ts,mjs,mts,cjs,cts}',
+      files: '*.{js,gjs,ts,gts,mjs,mts,cjs,cts}',
```

Comment:
Should we backport this to whatever version added the above plugin?

Reply:
hasn't been released yet

Reply:
(I think)